ddtrace/tracer/option.go

```diff
@@ -288,6 +288,12 @@ func SpanType(name string) StartSpanOption {
 	return Tag(ext.SpanType, name)
 }
 
+// MeasureSpan adds the measured key to a span, thereby making it eligible for
+// APM metrics/stats calculation.
+func MeasureSpan() StartSpanOption {
+	return Tag(keyMeasured, "1")
```

Comment:
We should make sure this ends up in the `.Metrics` map like all our internal stuff. Making it numeric should do the trick. Tests will confirm.
```suggestion
	return Tag(keyMeasured, 1)
```